Rudder/Classes/RudderConfig.m

```diff
@@ -15,41 +15,41 @@ - (instancetype)init
 {
     self = [super init];
     if (self) {
-        _endPointUrl = RudderBaseUrl;
+        _dataPlaneUrl = RudderBaseUrl;
```

Comment:
Can we change the name RudderBaseUrl to RudderDataPlaneUrl for better readability. Needs change in `Constants.m` file as well